modules/ROOT/pages/ip-whitelist.adoc

```diff
@@ -7,68 +7,66 @@ endif::[]
+* Any other origin
 
-. Enter one or more IP addresses in *Whitelist*.
-+
-image::ip-whitelist-051a0.png[height=353,width=377]
-+
-. Click +.
-+
-The greyed-out IP address and a deletion control appears below the Whitelist text entry field.
-+
-//image::ip-whitelist-1b671.png[height=353,width=377]
-image::ip-whitelist-1b671.png[height=475,width=500]
+=== Whitelisting Access Based on the IP address of the Request
+
+To whitelist access based on the IP address of the request, you can:
+
+* Define a specific IP address by enumerating it in the white space, for example, `192.168.1.1`.
+* Define a subset of addresses by identifying a subnet mask. 
+For example, `192.168.3.1/30`, which includes the consolidated range: `192.168.3.0 - 192.168.3.3`.
+* Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. 
 +
-. Click *Apply*.
+For example, setting `192.168` will include IP addresses from `192.168.0.0` to `192.168.255.255`.
+
+=== Whitelisting Access Based on the Origin IP Address of the x-Forwarded-For Header
+
+If the client connects to your API through an HTTP proxy or a load-balancer, you can blacklist the client's specific IP (the IP originating the request) instead of the address that appears in the request.
 
-== Whitelisting Based on the Origin IP Address of the x-Forwarded-For Header
+For example, suppose you want to whitelist `192.168.2.3` and the address of a client connecting through an HTTP proxy is `92.40.1.255`, the client requests appear with this public address using the proxy in such a case.
 
-If a client connects to an API through an HTTP proxy or a load-balancer, you might want to whitelist your client's specific IP, which is the IP originating the request, instead of the address that appears in the request. For example, if you want to whitelist 192.168.2.3 and this client is connecting through HTTP proxy 92.40.1.255, the request appears with the latter, public address. As a workaround, use the `X-Forwarded-For` header to identify the origin IP addresses of a request that was redirected to your endpoint.
+Typically, applications use the `X-Forwarded-For` header to identify the origin IP addresses of a request that was redirected to your endpoint.
 
-To whitelist the IP address of an originating request using `X-Forwarded-For` header:
+You can use a DataWeave 2.0 or MEL expression in the policy to instruct the service to look for the IP address in the `x-Forwarded-For` header. 
 
-. In Apply "IP whitelist" Policy, use a Mule expression in *IP Expression* to instruct the service to look for the originating IP address in the 'x-Forwarded-For' header of the Mule message:
+When you insert the IP address in the IP expression field of the policy parameters, the API Platform is instructed to look for the originating IP address that ranks in the concatenated values of the `x-Forwarded-For` header of the request:
```

Comment:
"for the originating IP address that ranks" ? Perhaps change ranks by the starting IP or similar?

Reply:
Yeah...I don't understand "...the originating IP address that ranks in the concatenated values of the..." either....

Reply:
Done! Thanks!